library/src/main/java/com/datatorrent/lib/io/fs/AbstractReconciler.java

```diff
@@ -73,9 +73,9 @@ public void process(INPUT input)
   // this stores the mapping from the window to the list of enqueued tuples
   private Map<Long, List<QUEUETUPLE>> currentWindowTuples = Maps.newConcurrentMap();
   private Queue<Long> currentWindows = Queues.newLinkedBlockingQueue();
-  private Queue<QUEUETUPLE> committedTuples = Queues.newLinkedBlockingQueue();
```

Comment:
Any reason why these are required to be protected?

I believe these fields are very important ones for correct functioning of Reconciler along with correct checkpointing.
By making these protected, we're exposing the overall functionality of of this class to get compromised.

In my opinion, we need a strong case for making this protected.